cli/src/test/java/com/devonfw/tools/ide/tool/graalvm/GraalVmTest.java

```diff
@@ -0,0 +1,87 @@
+package com.devonfw.tools.ide.tool.graalvm;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import com.devonfw.tools.ide.context.AbstractIdeContextTest;
+import com.devonfw.tools.ide.context.IdeTestContext;
+import com.devonfw.tools.ide.log.IdeLogLevel;
+
+/**
+ * Integration test of {@link GraalVm}.
+ */
+public class GraalVmTest extends AbstractIdeContextTest {
+
+  private static final String PROJECT_GRAALVM = "graalvm";
+
+  @Test
+  public void testGraalVmInstallInDirectoryExtra() {
+
+    // arrange
+    IdeTestContext context = newContext(PROJECT_GRAALVM);
+
+    GraalVm commandlet = new GraalVm(context);
+
+    // act
+    commandlet.run();
+
+    // assert
+    assertThat(context.getSoftwarePath().resolve("extra/graalvm/bin/HelloWorld.txt")).exists();
+    assertThat(context.getSoftwarePath().resolve("extra/graalvm/.ide.software.version")).exists().hasContent("22.3.3");
+  }
+
+  @Test
+  public void testAddTextToPropertiesFile() {
+
+    // arrange
+    IdeTestContext context = newContext(PROJECT_GRAALVM);
+    GraalVm commandlet = new GraalVm(context);
+
+    // act
+    commandlet.run();
+
+    // assert
+    String graalvmExport = "export GRAALVM_HOME=" + context.getSoftwarePath().resolve("extra/graalvm");
+    Path file = context.getConfPath().resolve("devon.properties");

```

Comment:
```suggestion
    Path file = context.getConfPath().resolve("ide.properties");
```

since now the variables are stored in `ide.properties`